admin/javascript/src/components/form-builder/index.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as schemaActions from 'state/schema/actions';
+import SilverStripeComponent from 'silverstripe-component';
+import FormComponent from 'components/form/index';
+import TextField from 'components/text-field/index';
+import HiddenField from 'components/hidden-field/index';
+import GridField from 'components/grid-field/index';
+import fetch from 'isomorphic-fetch';
```

Comment:
Might've been a bit too much future proofing, agreed. I don't see us server-side rendering SS4 views via NodeJS anytime soon. The important bit was using the new `fetch()` API standard, rather than `jQuery.ajax()`. How about we just use the https://github.com/github/fetch polyfill which `isomorphic-fetch` is based on?